.github/workflows/test-functions.yml

```
@@ -0,0 +1,16 @@
+name: Test Functions Snippets
+
+on:
+  pull_request:
+    paths:
+      - "examples/functions/**"
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 16.x
+      - run: cd examples/functions/functions_tester && npm ci && npm run test -- --forceExit
```

```suggestion
      - run: cd examples/functions/functions_tester && npm ci && npm run test --forceExit
```

pretty sure you don't need the `--` here. you only need that if you want to specify a path to files.

---

As discussed on slack, the `--` is needed to denote passing the subsequent args directly to the inner command (`jest`) as opposed to the outer command (`npm run`)